pkgs/top-level/all-packages.nix

```diff
@@ -26614,6 +26614,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Carbon;
   };
 
+  cyan = callPackage ../applications/graphics/cyan {};
```

Comment:
```suggestion
  cyan = libsForQt5.callPackage ../applications/graphics/cyan {};
```
If this is a native QT application maybe we should call it like this? We could remove the qt5 prefix above then.

Reply:
I recommend not to use `libsForQt5`, because it is incompatible with the new rec-less `stdenv.mkDerivation`.

Reply:
If it is a qt5 application requiring qt5 frameworks it must use it.

Reply:
> I recommend not to use `libsForQt5`, because it is incompatible with the new rec-less `stdenv.mkDerivation`.

That is not even used here. qt things are meant to be used with libsForQt5 and now probably another end is not fully working.